rosdep/python.yaml

```
@@ -5268,6 +5268,15 @@ python3:
   gentoo: [dev-lang/python]
   rhel: ['python%{python3_pkgversion}-devel']
   ubuntu: [python3-dev]
+python3-argcomplete:
+  alpine: [py3-argcomplete]
+  arch: [python-argcomplete]
+  centos: [python3-argcomplete]
+  debian: [python3-argcomplete]
+  fedora: [python3-argcomplete]
+  freebsd: [py37-argcomplete]
+  gentoo: [dev-python/argcomplete]
```

rosdep aliases `centos` to `rhel`, so we should use that instead. Also, the package for RHEL 7 comes from EPEL and is called `python36-argcomplete`. We can use the `%{python3_pkgversion}` RPM macro to resolve to the system default package suffix, `36` on RHEL 7 and `3` on RHEL 8.

```suggestion
  debian: [python3-argcomplete]
  fedora: [python3-argcomplete]
  freebsd: [py37-argcomplete]
  gentoo: [dev-python/argcomplete]
  rhel: ['python%{python3_pkgversion}-argcomplete']
```